parcel_test.go

```diff
@@ -31,57 +32,109 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	// настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	require.NoError(t, err)
+	defer db.Close()
+
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
 	// add
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	newId, err := store.Add(parcel)
+	require.NoError(t, err)
+	require.NotEmpty(t, newId)
 
 	// get
 	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
 	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+	newParcel, err := store.Get(newId)
+	require.NoError(t, err)
+
+	assert.Equal(t, parcel.Client, newParcel.Client)
+	assert.Equal(t, parcel.Status, newParcel.Status)
+	assert.Equal(t, parcel.Address, newParcel.Address)
+	assert.Equal(t, parcel.CreatedAt, newParcel.CreatedAt)
 
 	// delete
 	// удалите добавленную посылку, убедитесь в отсутствии ошибки
 	// проверьте, что посылку больше нельзя получить из БД
+	err = store.Delete(newId)
+	require.NoError(t, err)
+	_, err = store.Get(newId)
+	assert.Equal(t, sql.ErrNoRows, err)
 }
 
 // TestSetAddress проверяет обновление адреса
 func TestSetAddress(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	// настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	require.NoError(t, err)
+	defer db.Close()
+
+	store := NewParcelStore(db)
+	parcel := getTestParcel()
 
 	// add
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	newId, err := store.Add(parcel)
+	require.NoError(t, err)
+	require.NotEmpty(t, newId)
 
 	// set address
 	// обновите адрес, убедитесь в отсутствии ошибки
 	newAddress := "new test address"
+	err = store.SetAddress(newId, newAddress)
+	require.NoError(t, err)
 
 	// check
 	// получите добавленную посылку и убедитесь, что адрес обновился
+	newParcel, err := store.Get(newId)
+	require.NoError(t, err)
+	assert.Equal(t, newAddress, newParcel.Address)
```

Comment:
Для сравнения ошибок нужно использовать специальную функцию
```suggestion
	assert.ErrorIs(t, err, sql.ErrNoRows)
```